LearningMissionSimulation/PlaygroundGavril.cs

```diff
@@ -47,19 +59,30 @@ public  void CreateAccounts(int accountCount)
                     }
                 }
             }
+
+            ReportSummary(actionName: action, itemCount: accountCount);
+            ReportFooter(actionName: action);
         }
 
         public void ActivateAccounts()
         {
+            string action = "Activate account";
+            ReportHeader(actionName: action);
+            int i = 0;
             foreach (var account  in pendingAccountList )
             {
                 account.Status = Status.Active; 
                 pendingAccountList.Remove(account);
+                ReportItem(itemName: account.ToString(), actionName: action, itemIndex: i);
```

Comment:
Index i is always 0.

Reply:
Adding i++ on line 85 is not fixing the issue. You should increment the index inside the loop.

Reply:
ES dreci i++ cikli nersum.

Reply:
It is NOT inside the loop, please fix it.

Reply:
DONE

Reply:
DONE